jota/src/main/java/org/iota/jota/IotaAPI.java

```diff
@@ -1628,7 +1626,6 @@ public boolean isPromotable(Transaction tail) throws ArgumentException {
      * @return <tt>true</tt> if it is, otherwise <tt>false</tt>
      * @throws ArgumentException when we can't get the consistency of this transaction
```

Comment:
This method still throws it, but replace explanation to "When we cant find the transaction"